docs/deployment-examples.md

```diff
@@ -0,0 +1,331 @@
+ - [Executable Binary Deployment](#executable-binary-deployment)
+    - [Binary Specific Prerequisites](#binary-specific-prerequisites)
+    - [Binary - default - http passing API token with query](#binary---default---http-passing-api-token-with-query)
+    - [Binary - Tokens File - http with API tokens file embedded in exporter](#binary---tokens-file---http-with-api-tokens-file-embedded-in-exporter)
+    - [Binary - TLS - https passing API token with query](#binary---tls---https-passing-api-token-with-query)
+
+# Prerequisites
+All deployments will require an API token to authenticate with the array. Read-only only user access is recommended.
+
+1. Generate an API token from your chosen user account or create a new readonly user.
+    API token can be retrieved from either Purity GUI ot CLI.
+
+    ```console
+    pureuser@array01> pureadmin create --role readonly svc-readonly
+    Name          Type   Role    
+    svc-readonly  local  readonly
+
+    pureuser@array01> pureadmin create --api-token svc-readonly
+    Name          Type   API Token                             Created                  Expires
+    svc-readonly  local  a1b2c3d4-e5f6-1234-5678-a1b2c3d4e5f6  2023-11-02 12:00:00 EST  -      
+    ```
+
+# Container Deployment
+## Container - default - http passing API token with query
+We build container images and publish them to RedHat quay.io. Here they can be continually scanned for vulnerabilities and updated as soon as a new version are available.
+You can navigate to [https://quay.io/repository/purestorage/pure-fa-om-exporter](https://quay.io/repository/purestorage/pure-fa-om-exporter) to view the available versions. In this example we will pull a container down using Docker.
+
+In this example we will use the default port 9490, set the name as pure-fa-om-exporter and pull the latest version of the image from quay.io/purestorage/pure-fa-om-exporter. The `--detach` switch sends the process to the background.
+
+1. **Run and pull the image**
+    ```console
+    $ docker run --detach --publish 9490:9490 --name pure-fa-om-exporter --restart unless-stopped quay.io/purestorage/pure-fa-om-exporter:latest
+    ```
+    In this guide we will not be covering Docker troubleshooting.
+
+2. **Check the container is running**
+    ```console
+    $ docker ps
+    CONTAINER ID   IMAGE                                            COMMAND                  CREATED         STATUS         PORTS                                       NAMES
+    0076ded8a073   quay.io/purestorage/pure-fa-om-exporter:latest   "/pure-fa-om-exporte…"   10 seconds ago   Up 10 seconds   0.0.0.0:9490->9490/tcp, :::9490->9490/tcp   pure-fa-om-exporter
+    ```
+
+3. **Test the exporter**
+    Use `curl` to test the exporter returns results.
+    ```console
+    $ curl -H 'Authorization: Bearer a1b2c3d4-e5f6-1234-5678-a1b2c3d4e5f6' -X GET 'http://localhost:9490/metrics/array?endpoint=array01' -silent | grep ^purefa_info
```

Comment:
instead of an actual token use a placeholder such as `<API token>`. 
Use a placeholder for the array name as well.